protos/sigstore_trustroot.proto

```diff
@@ -117,3 +118,50 @@ message TrustedRoot {
         // A set of trusted timestamping authorities.
         repeated CertificateAuthority timestamp_authorities = 5;
 }
+
+
+// SigningConfig represents the trusted entities/state needed by Sigstore
+// signing. In particular, it primarily contains service URLs that a Sigstore
+// signer may need to connect to for the online aspects of signing.
+message SigningConfig {
+        // A URL to a Fulcio-compatible CA, capable of receiving
+        // Certificate Signing Requests (CSRs) and responding with
+        // issued certificates.
+        //
+        // This URL **MUST** be the "base" URL for the CA, which clients
+        // should construct an appropriate CSR endpoint on top of.
+        // For example, if `ca_url` is `https://example.com/ca`, then
+        // the client **MAY** construct the CSR endpoint as
+        // `https://example.com/ca/api/v2/signingCert`.
+        string ca_url = 1;
```

Comment:
Aligning with the above names seems better for readability, so +1 with what's here.

I did bring up a related [issue](https://github.com/sigstore/sigstore-go/issues/71) in sigstore-go where I suggested we call tlog related functions `RekorLogs` as Rekor is the only valid log. I think the same argument could be applied to Fulcio given we couldn't support a generic CA without standardization on request payload and headers. I could see an argument to calling these `(rekor|fulcio)_url`, though we ideally would also change the above to `rekor_logs`/`fulcio_cas`. Given I don't think we should do a breaking change just for names, I'd stick with `(ca|tlog)_url`.

Reply:
Thanks!